docs/releases/5.0.txt

```diff
@@ -37,6 +37,13 @@ compatible with Django 5.0.
 What's new in Django 5.0
 ========================
 
+Facet filters in the admin
+--------------------------
+
+Facet counts are now shown for applied filters in admin change-list UI. By
+default, facet counts can be toggled via a ``_facets`` query parameter, but
+this behavior can be configured via the new :attr:`.ModelAdmin.show_facets`.
+
```

Comment:
I'd add a reference to the `Facets` section in the `docs/ref/contrib/admin/filters.txt`